packages/ui/src/modal/sweep/SweepModalRenderer.tsx

```diff
@@ -424,34 +381,35 @@ export const SweepModalRenderer: FC<Props> = ({
       partial: true,
     }
 
-    if (referrer && referrerFeeBps) {
-      const price = toFixed(total, currency?.decimals || 18)
-      const fee =
-        (Number(parseUnits(`${Number(price)}`, currency?.decimals || 18)) *
-          referrerFeeBps) /
-        10000
-      const atomicUnitsFee = formatUnits(BigInt(fee), 0)
-      options.feesOnTop = [`${referrer}:${atomicUnitsFee}`]
-    } else if (referrer && referrerFeeFixed) {
-      options.feesOnTop = [`${referrer}:${referrerFeeFixed}`]
-    } else if (referrer === null && referrerFeeBps === null) {
+    if (feesOnTopFixed && feesOnTopFixed.length > 0) {
```

Comment:
feesOnTopBps should take precedent over feesOnTopFixed to maintain consistency with the BuyModal